src/components/Cards/MinutesItemCard/MinutesItemCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from "react";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface Docs {
+  /**Document item label */
+  label: string;
+  /**Document item url */
+  url: string;
+}
+
+interface Items {
+  /**Minute item headline */
+  item: string;
+  /**Array of documents to be shown below */
+  docs?: Docs[];
+}
+
+export interface MinutesItemProps {
+  minutesItems: Items[];
+}
+
+function returnDocList(docs: Docs[] | undefined) {
+  if (docs !== undefined) {
+    return (
+      <section>
+        <details>
+          <summary style={{ color: "blue" }}>
+            <p>See documents</p>
+          </summary>
```

Comment:
Only nitpick for me is I think making this be black text with an underline may be better. @hawkticehurst @tohuynh thoughts?

Reply:
Which, we have `cdp-design` that already has this style definition. @tohuynh I know we used `cdp-design` in this repo already [here](https://github.com/CouncilDataProject/cdp-frontend/blob/main/src/components/Cards/PersonCard/PersonCard.tsx#L88) but I don't see the css import.

However we handled that, @andrewlubrino you _should_ be able to just use `cdp-link`.

Reply:
For some reason, you don't need to import the protocol.css and cdp-design css in order to get the styling.

Ideally, we are supposed to import any global css in .storybook/preview.js. This isn't currently the case for protocol and cdp-design, but the styling still gets applied anyway. I have no clue what's happening though.

Also there isn't a cdp-link color in cdp-design repo.

Reply:
Agree about black for `See documents`, I think blue is reserved for links.

Reply:
Applying the `cdp-link` class didn't give me black text with an underline, so I just styled the `<summary>` element and its child `<p>` using the emotion/style that To mentioned in the comments on the Footer component. I was able to move the "+" closer to "See documents" by changing the width of the `<summary>`  element to 120px. Is this okay? I'm just wondering if that won't scale properly.